pkg/component/networking/istio/charts/istio/istio-ingress/templates/proxy-protocol-envoyfilter.yaml

```diff
@@ -25,4 +25,65 @@ spec:
         - name: proxy_protocol
           typed_config:
             "@type": "type.googleapis.com/envoy.extensions.filters.listener.proxy_protocol.v3.ProxyProtocol"
-{{ end -}}
\ No newline at end of file
+        - name: inner_proxy_protocol
+          typed_config:
+            "@type": "type.googleapis.com/envoy.extensions.filters.listener.proxy_protocol.v3.ProxyProtocol"
+            allow_requests_without_proxy_protocol: true
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  labels:
+{{ .Values.labels | toYaml | indent 4 }}
+  name: proxy-protocol-9443
+  namespace: {{ .Release.Namespace }}
+spec:
+  workloadSelector:
+    labels:
+{{ .Values.labels | toYaml | indent 6 }}
+  configPatches:
+  - applyTo: LISTENER
+    match:
+      context: GATEWAY
+      listener:
+        portNumber: 9443
+    patch:
+      operation: MERGE
+      value:
+        listener_filters:
+        - name: proxy_protocol
+          typed_config:
+            '@type': type.googleapis.com/envoy.extensions.filters.listener.proxy_protocol.v3.ProxyProtocol
+            allow_requests_without_proxy_protocol: true
+        - name: envoy.filters.listener.tls_inspector
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.listener.tls_inspector.v3.TlsInspector
+        per_connection_buffer_limit_bytes: 32768
```

Comment:
Why is this necessary? How does this relate to the `istio-ingressgateway` envoy filter in file `envoy-filter.yaml?